src/Tools/babylon.assetsManager.ts

```diff
@@ -186,8 +186,6 @@ module BABYLON {
     }
 
     export interface ITextureAssetTask extends IAssetTask {
-        onSuccess: (task: ITextureAssetTask) => void;

```

Comment:
Just for me to understand - is this also related to ts 2.6?

Reply:
Yes. Typescript complains that the definition of onSuccess is not identical to the definition of onSuccess in IAssetTask. (if strictFunctionTypes is enabled)